### ISSUE TITLE
click "Home" on sidebar, url turn to be "http://localhost:4000/%20/"

### ISSUE THREAD

**Issue by wangyangang** [LouisBarranqueiro/hexo-theme-tranquilpeak#589]
### Description
when i click the home button on sidebar, web browser go to "http://localhost:4000/%20/",
other button on sidebar work fine but home button.

### Expected behavior
web browser go to "http://localhost:4000/" or "http://localhost:4000"

### Steps to reproduce the bug
 I follow your documentation to config this theme on my website,
it occur.

### Environment
 - **Operating system with version** : Macos catalina 10.15.1
 - **Node version**: v10.16.0
 - **Hexo version**: 4.0.0
 - **Hexo-cli version**: 2.0.0

### Additional information
- [* ] I can fix this bug 
by edit _config.yml file:
```
sidebar:
    menu:
        home:
            title: global.home
            # url: /            # change this line with next line.
            url: ""
```

**Comment by LouisBarranqueiro** [comment]
👋 @wangyangang, thanks for reporting this bug. I will release a new version soon containing a fix for it.

**Comment by chrisjwelly** [comment]
Facing the same issue on:
Ubuntu 19.10
Hexo 4.0.0
Hexo-cli 3.1.0
Node 12.3.1

The fix also works for me. 

Thanks for the hard work! @LouisBarranqueiro

**Comment by LouisBarranqueiro** [comment]
👋 @chrisjwelly, I will release this new version probably tonight or tomorrow :)

**Comment by darekkay** [comment]
@chrisjwelly As a workaround (or a solution?), you can change the `' '` into `''` [here](https://github.com/LouisBarranqueiro/hexo-theme-tranquilpeak/blob/master/layout/_partial/sidebar.ejs#L36).

**Comment by wgspring** [comment]
@LouisBarranqueiro I facing the same issue too. And 
there is a similar issue:
when I click the title of each post page, it also turn to be "http://localhost:4000/%20/".

I find a solution for these issue:
change the ' ' into '' [here](https://github.com/LouisBarranqueiro/hexo-theme-tranquilpeak/blob/master/layout/_partial/sidebar.ejs#L36).  for issue of home button.
change the ' ' into '' [here](https://github.com/LouisBarranqueiro/hexo-theme-tranquilpeak/blob/master/layout/_partial/header.ejs#L19).  for issue of title bar.

**Comment by chilo-hzl** [comment]
@wgspring You are so nice, i encountered this issue long time, i don't know how to work out it, even i forgot to see github issue, today i finally resolve this, thank you so much

**Comment by LouisBarranqueiro** [comment]
👋 guys, the new version has been released: [v4.0.0](https://github.com/LouisBarranqueiro/hexo-theme-tranquilpeak/releases/tag/v4.0.0). Sorry for the delay.

**Comment by sj602** [comment]
I think the bug still exists even after the v4.0.0 update.
To fix, erase ' ' => ''
header.ejs
```
            href="<%- url_for('') %>"

```
